.github/workflows/doc-release.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow builds and deploys documentation on a release.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+
+name: Doc Deployment
+
+on:
+  release:
+    types: [published]
+
+
+jobs:
+  build:
+    name: Build and Upload Site Artifact
+    
+    runs-on: ubuntu-latest
+
+    env:
+      python-version: "3.11"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Set up Python ${{ env.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ env.python-version }}
+      
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade -r docs/user/requirements.txt
+      
+      - name: Build Documentation
+        run: |
+          mkdocs build --strict
+      
+      - name: Upload Github Pages Site Artifact
+        uses: actions/upload-pages-artifact@v1
+        with:
+          name: 'github-pages'
+          path: "site/"
```

Comment:
Is there a pattern to the different quote styles being used? You don't need to change it, just curious why two simple strings close together have single and double quotes.

Reply:
Updated. They should match.